build_distribution.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Build UI
+cd concord-ui
+npm run build
+cp -rf build ../concord-server/server/src/main/resources/ui
```

Comment:
May have to clear previous files in /concord-server/server/src/main/resources/ui folder